tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -54,18 +54,20 @@ function makeAllTheProviders({context, organization}: ProviderOptions) {
 }
 
 /**
- * Migrating from enzyme? Pass context via the options object
- * Before
- * mountWithTheme(<Something />, routerContext);
- * After
- * mountWithTheme(<Something />, {context: routerContext});
+ * Migrating from enzyme?
+ * As a rule of thumb, try avoiding unnecessary context and just mount your component.
+ * If it works, then congratulations, you dont need anything else.
+ * mountWithTheme(<TestedComponent />);
```

Comment:
should this be `render(<TestedComponent />);` since `mountWithTheme` is renamed and exported as `render`?

Reply:
Yes, I updated the comment in another PR, but I will update it here too. This is still an RFC btw 😅

Reply:
gotcha 👍

Reply:
@udameli good catch though, we need to update this if we ever merge so we make sure not to override it :)